database/src/main/public/global_id.sql

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+--DROP SEQUENCE IF  EXISTS public.global_id_seq
+
+-- SERVER should be a unique number within your deployment between 0 and 9222
+CREATE SEQUENCE IF NOT EXISTS public.global_id_seq
+    INCREMENT 1
+    START [SERVER_ID]*1000000000000000+1
+    MINVALUE [SERVER_ID]*1000000000000000+1
+    MAXVALUE [SERVER_ID] + 1)*1000000000000000
+    CACHE 1;
```

Comment:
Forgive my ignorance, but what is the point of this?

Reply:
This code is not directly executable (obviously). But it suggests, how to make the ID unique - within our realm. Each database would have it's number giving a unique range and therefore ids across all out databases.

Of course UUID could be used as well for IDs instead, happy to discuss what would be better for Enceladus.
@Zejnilovic @AdrianOlosutean

Reply:
At a first glance I think I prefer UUID

Reply:
Well, this does seem very interesting. I can see a few nice things coming out of it and no drawbacks so far, so IMHO we can keep it.

Reply:
BIGINT
* little more hassle to install with it's global property
* little more human friendly to read, imho
* half the size
* maybe tad bit faster

UUID 
* somewhat common to use nowadays
* natural real global unique
* no need to put sequence changes to commit log (no IO for the ID generation needed)

None of the points make nor break the candidate. So really question of our preference.